lua/jmod/cl_init.lua

```diff
@@ -377,10 +377,9 @@ net.Receive("JMod_LuaConfigSync", function(dataLength)
 	JMod.LuaConfig = JMod.LuaConfig or {}
 	JMod.LuaConfig.ArmorOffsets = Payload.ArmorOffsets
 	JMod.Config = JMod.Config or {}
-	JMod.Config.General = JMod.Config.General or {}
-	JMod.Config.Weapons = JMod.Config.Weapons or {}
-	JMod.Config.General.AltFunctionKey = Payload.AltFunctionKey
-	JMod.Config.Weapons.SwayMult = Payload.WeaponSwayMult
+	JMod.Config.General = {["AltFunctionKey"] = Payload.AltFunctionKey}
+	JMod.Config.Machines = {["Blackhole"] = Payload.Blackhole}
+	JMod.Config.Weapons = {["SwayMult"] = Payload.WeaponSwayMult}

```

Comment:
Why were square brackets and quote marks used?
Wouldn't `{AltFunctionKey = PayLoad.AltFunctionKey}` work?
Or is this a workaround to a problem?

Reply:
cause its a index of the array, don't you have to use [x] to indicate that?

Reply:
Nvm, ig you don't. I saw us doing that somewhere else so assumed it was necessary.